src/ray/raylet/node_manager.cc

```diff
@@ -551,7 +563,7 @@ void NodeManager::HandleRestoreSpilledObject(
 void NodeManager::HandleReleaseUnusedBundles(
     const rpc::ReleaseUnusedBundlesRequest &request,
     rpc::ReleaseUnusedBundlesReply *reply, rpc::SendReplyCallback send_reply_callback) {
-  RAY_LOG(DEBUG) << "Releasing unused bundles.";
+  RAY_LOG(INFO) << "Releasing unused bundles.";
```

Comment:
Can you add more information? This is meaningless to only log this for a normal user perspective. We can probably 
1. why we release unused bundles.
2. Which bundles were released